jupyter-book/linear_models/linear_models_quiz_02.md

```diff
@@ -1,7 +1,7 @@
 # ✅ Quiz
 
 ```{admonition} Question
-Is it possible to fit a linear model on a dataset where the data have a
+Is it possible to fit a linear model on a dataset where the data has a
```

Comment:
So some people insist `data` is plural (datum is the singular). Honestly I couldn't care less about this, I think we should leave it as is, there are higher priorities.

And sorry since I couldn't care less I don't even think we try to make this consistent in the MOOC